server/src/main/java/org/elasticsearch/search/DefaultSearchContext.java

```diff
@@ -156,6 +157,8 @@ final class DefaultSearchContext extends SearchContext {
             engineSearcher.getQueryCachingPolicy(),
             minimumDocsPerSlice,
             lowLevelCancellation,
+            // TODO not set the for now, this needs a special thread pool and can be enabled after its introduction
+            // parallelize ? (EsThreadPoolExecutor) this.indexService.getThreadPool().executor(ThreadPool.Names.AUTO_COMPLETE) : null
```

Comment:
just to be completely sure, can you replace AUTO_COMPLETE with something that does not compile? :)

Reply:
But leave it commented out I guess? Or make it so the PR doesn't compile?

Reply:
leave it commented out. The plan is to merge your PR, and that's not possible if it does not compile :)